dali/operators/generic/transpose/transpose.h

```diff
@@ -22,7 +22,6 @@
 #include <utility>
```

Comment:
I think you can remove cuTT from the Acknowledgements.txt

Reply:
Done.